gazelle/README.md

```diff
@@ -438,7 +504,7 @@ for more information on extending Gazelle.
 
 If you add new Go dependencies to the plugin source code, you need to "tidy" the go.mod file.
 After changing that file, run `go mod tidy` or `bazel run @go_sdk//:bin/go -- mod tidy`
-to update the go.mod and go.sum files. Then run `bazel run //:update_go_deps` to have gazelle
+to update the go.mod and go.sum files. Then run `bazel run //:gazelle_update_repos` to have gazelle
```

Comment:
I wasn't able to run `//:update_go_deps` so I changed it to what worked for me. Please LMK if it's incorrect.

Though note that I _didn't actually update all go deps_.